LokiLabs/microsoft-graph-explorer-teams-app
# AB#39541 Interns/gracet9182/39541 sample queries

# Goal
* Follow mock up of creating a new section for sample queries before the query runner
* Pull sample queries from the devxapi api
* Filter through the relevant queries depending on if you are in chats or teams (i.e. you only want to see chat sample queries when you attach it to a chat instance)
* When you click any of the queries, it can fill the query runner for convenience 
* When you click on the button, it will open a new page for rendering the documentation page
* Note that this is dependent on #33 because it requires the query runner to work
# Test
* Pull the branch
* Click on the sample query buttons to see it being retrieved and put in the query runner


https://user-images.githubusercontent.com/47228306/128271036-1ec64565-8485-44b9-ae57-9582d06361a7.mp4
